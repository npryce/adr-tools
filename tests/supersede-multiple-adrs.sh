adr new First Record
adr new Second Record
adr new -s 1 -s 2 Third Record
head -8 doc/adr/0001-first-record.md
head -8 doc/adr/0002-second-record.md
head -12 doc/adr/0003-third-record.md
