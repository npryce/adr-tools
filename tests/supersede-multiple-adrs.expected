adr new First Record
doc/adr/0001-first-record.md
adr new Second Record
doc/adr/0002-second-record.md
adr new -s 1 -s 2 Third Record
doc/adr/0003-third-record.md
head -8 doc/adr/0001-first-record.md
# 1. First Record

Date: 1992-01-12

## Status

Superseded by [3. Third Record](0003-third-record.md)

head -8 doc/adr/0002-second-record.md
# 2. Second Record

Date: 1992-01-12

## Status

Superseded by [3. Third Record](0003-third-record.md)

head -12 doc/adr/0003-third-record.md
# 3. Third Record

Date: 1992-01-12

## Status

Accepted

Supersedes [1. First Record](0001-first-record.md)

Supersedes [2. Second Record](0002-second-record.md)

